cmd/orb.go

```diff
@@ -366,17 +369,38 @@ func logOrbs(orbCollection *api.OrbsForListing) error {
 	return nil
 }
 
+var validSortFlag = map[string]bool{
+	"builds":   true,
+	"projects": true,
+	"orgs":     true}
+
+func validateSortFlag(sort string) error {
+	if _, valid := validSortFlag[sort]; valid {
+		return nil
+	}
+	// TODO(zzak): we could probably reuse the map above to print the valid values
+	return fmt.Errorf("expected `%s` to be one of \"builds\", \"projects\", or \"orgs\"", sort)
+}
+
 func listOrbs(opts orbOptions) error {
 	if len(opts.args) != 0 {
 		return listNamespaceOrbs(opts)
 	}
 
-	orbs, err := api.ListOrbs(opts.apiOpts, orbListUncertified)
+	orbs, err := api.ListOrbs(opts.apiOpts, opts.listUncertified)
 	if err != nil {
 		return errors.Wrapf(err, "Failed to list orbs")
 	}
 
-	return logOrbs(orbs)
+	if opts.sortBy != "" {
+		if err := validateSortFlag(opts.sortBy); err != nil {
```

Comment:
Nit: The validation check would return a little faster if it ran before the API request `api.ListOrbs(...)`